plugins/catalog-backend-module-bitbucket-cloud/src/service/BitbucketCloudCatalogModule.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  configServiceRef,
+  createBackendModule,
+  loggerServiceRef,
+  loggerToWinstonLogger,
+  schedulerServiceRef,
+} from '@backstage/backend-plugin-api';
+import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node';
+import { BitbucketCloudEntityProvider } from '../BitbucketCloudEntityProvider';
+
+/**
+ * Bitbucket Cloud module for the Catalog plugin
+ *
+ * @alpha
+ */
+export const bitbucketCloudCatalogModule = createBackendModule({
+  moduleId: 'bitbucketCloud',
+  pluginId: 'catalog',
+  register(env) {
+    // plugins depending on this API will be initialized before this plugins init method is executed.
```

Comment:
maybe not necessary to have this comment?

Reply:
basically, copy and paste from another plugin and I just kept it. But sure, I will just remove it.